internal/app/kdk/destroy.go

```diff
@@ -0,0 +1,66 @@
+// Copyright © 2018 Cisco Systems, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package kdk
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/client"
+	"github.com/manifoldco/promptui"
+)
+
+func Destroy(ctx context.Context, dockerClient *client.Client, logger logrus.Entry) error {
+
+	var containerIds []string
+
+	containers, err := dockerClient.ContainerList(ctx, types.ContainerListOptions{})
+
+	if err != nil {
+		logger.WithField("error", err).Fatal("Failed to list docker containers")
+	}
+	for _, container := range containers {
+		for _, name := range container.Names {
+			if strings.HasPrefix(name, "/kdk") {
```

Comment:
Will destroy kill ALL kdk's?  You showed me a demo where you could launch multiple kdk's.

Reply:
It will destroy all containers with `kdk` prefix.  Maybe we should add a `prefix string` arg to `Destroy`.

Reply:
You specifically name the kdk in the config.yaml.  Can we pick it out by name what to destroy?

Regardless, this is a should and not a must.   We can followup a pr later for multi-kdk support.

Reply:
Perhaps  destroy can pattern match on the global "Name"

Reply:
IDK... I'd like to be able to destroy all `kdk` containers regardless of config.

also, notice that user is prompted for confirmation before deletion